src/device/simulator.ts

```diff
@@ -163,6 +175,10 @@ export class SimulatorDeviceConnection
       }
       case "request_flash": {
         this.emit(EVENT_REQUEST_FLASH);
+        this.logging.event({
+          type: "sim-user",
+          message: "start",
```

Comment:
"start" or "play"?

Reply:
Let's go for start.

Let's make sure we update the docs.